sdk/go/common/workspace/plugins.go

```diff
@@ -809,6 +809,81 @@ func (info PluginInfo) Install(tgz io.ReadCloser, reinstall bool) error {
 	return info.InstallWithContext(context.Background(), tarPlugin{tgz}, reinstall)
 }
 
+// DownloadToFile downloads the given PluginInfo to a temporary file and returns that temporary file.
+// This has some
```

Comment:
Nit: incomplete comment?

Reply:
Huh, must of context switched halfway through writing that.